### PR TITLE
xformers installation fix

### PR DESCRIPTION
**Pull request by santiatpml**

### CODE REVIEW

File: pgml-extension/requirements.txt [hunk 1]
```diff
@@ -18,7 +18,6 @@ torchaudio==2.0.2
 torchvision==0.15.2
 tqdm==4.65.0
 transformers==4.29.2
-xformers==0.0.20
```
Comment (levkk) [inline pgml-extension/requirements.txt:21]: We use this file to install dependencies elsewhere, e.g. in our production cloud. Would it be possible to add another file, perhaps `requirements-xformers.txt` to install that special package?

Comment (levkk) [reply pgml-extension/requirements.txt:21]: I think you can do:

_requirements.txt_
```
xformers==0.0.20 --no-dependencies
```

which is then installable with 

```
pip install -r requirements-xformers.txt
```

Comment (levkk) [reply pgml-extension/requirements.txt:21]: I'm afraid that if we put this package only in the dockerfile, we'll forget to add it elsewhere where it's needed.

Comment (santiatpml) [reply pgml-extension/requirements.txt:21]: `--no-dependencies` has to be outside requirements.txt file.
`pip install -r requirements-xformers.txt --no-dependencies`

**Pull request merged**